### ISSUE TITLE
Error [ERR_REQUIRE_ESM]: require() of ES Module when using lb4 app

### ISSUE THREAD

**Issue by lucilejerber** [loopbackio/loopback-next#9393]
### Describe the bug

Hello, 
I am trying to use loopback for the first time. I followed the [Getting started guide](https://loopback.io/doc/en/lb4/Getting-started.html). However, after I created the app through ` lb4 app`, when I run `npm start`, I get this error :
`Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/jerber/ui-back/node_modules/lodash-es/lodash.js from /Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js not supported.`

### Logs

```shell
npm start

> ui-back@0.0.1 prestart
> npm run rebuild


> ui-back@0.0.1 rebuild
> npm run clean && npm run build


> ui-back@0.0.1 clean
> lb-clean dist *.tsbuildinfo .eslintcache


> ui-back@0.0.1 build
> lb-tsc


> ui-back@0.0.1 start
> node -r source-map-support/register .


/Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js:8
const lodash_es_1 = require("lodash-es");
                    ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/jerber/ui-back/node_modules/lodash-es/lodash.js from /Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js not supported.
Instead change the require of lodash.js in /Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js:8:21)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/convert.js:7:34)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/index.js:8:19)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@loopback/rest/dist/validation/request-body.validator.js:14:22)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@loopback/rest/dist/parser.js:15:34)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@loopback/rest/dist/index.js:32:22)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@loopback/rest-explorer/dist/rest-explorer.component.js:10:16)
    at Object.<anonymous> (/Users/jerber/ui-back/node_modules/@loopback/rest-explorer/dist/index.js:18:22)
    at Object.<anonymous> (/Users/jerber/ui-back/dist/application.js:6:25)
    at Object.<anonymous> (/Users/jerber/ui-back/dist/index.js:6:23)
```

### Additional information

```
node -v
v18.15.0
```

```
npm -v
9.5.0
```

### Reproduction

https://loopback.io/doc/en/lb4/Getting-started.html

**Comment by channa-zoomi** [comment]
I am working with lb4 more than 2 years. But today I got same issue in my projects as described above.

Successfully build the projects, but I am getting error as below when I am trying to run the project with 'npm run start'

`/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js: 8
const lodash_es_1 = require("lodash-es");
                    ^

Error [ERR_REQUIRE_ESM
]: require() of ES Module /Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/lodash-es/lodash.js from /Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js not supported.
Instead change the require of lodash.js in /Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/converters/schema.js: 8: 21)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/lib/convert.js: 7: 34)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@openapi-contrib/openapi-schema-to-json-schema/dist/cjs/index.js: 8: 19)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/rest/dist/validation/request-body.validator.js: 14: 22)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/rest/dist/parser.js: 15: 34)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/rest/dist/index.js: 32: 27)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/authentication/dist/providers/auth-action.provider.js: 10: 16)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/authentication/dist/providers/index.js: 8: 27)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/authentication/dist/authentication.component.js: 11: 21)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/node_modules/@loopback/authentication/dist/index.js: 24: 27)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/dist/application.js: 5: 26)
    at Object.<anonymous> (/Users/myuser/work/main/test/project/project-backend/project-nodejs/dist/index.js: 7: 23) {
  code: 'ERR_REQUIRE_ESM'
}
`

**Comment by patrioticcow** [comment]
This exact thing started happening to me today around 1pm PST
Tested on node 16 and 18

**Comment by raymondfeng** [comment]
Duplicate of https://github.com/openapi-contrib/openapi-schema-to-json-schema/issues/54

**Comment by shubhamp-sf** [comment]
The cause of the issue is related to [this commit](https://github.com/openapi-contrib/openapi-schema-to-json-schema/commit/bdac387ce9901898e59c41fa4d2c2041e938d474) on the GitHub repository of the openapi-schema-to-json-schema package, marked as BREAKING. However, despite being a major update, it was not released as a new version, which has resulted in the package being automatically upgraded for all users. This issue has been reported on the package's repository as @raymondfeng mentioned above. As a solution, one could use an override in their package.json file to specify a specific version, such as "3.2.0", of the package to use instead.

```json
"overrides": {
    "@openapi-contrib/openapi-schema-to-json-schema": "3.2.0"
}
```

**Comment by Ojvar** [comment]
I append 
"overrides": {
    "@openapi-contrib/openapi-schema-to-json-schema": "3.2.0"
}
to package.json
But I still receive the error yet  :(((

**Comment by Ojvar** [comment]
Thanks
It works ;)

**Comment by shubhamp-sf** [comment]
They just published a fix in `v3.3.1`, can anyone confirm?

**Comment by Ojvar** [comment]
I run this
npm i -g @loopback/cli  
agian
after that I crate a fresh app
and booooooooom, it worked correctly

**Comment by shubhamp-sf** [comment]
Thanks @Ojvar for testing this out. 🎉

**Comment by jonluca** [comment]
Apologies everyone, looks like my semantic release commit text knowledge was a bit out of date.

The esm/cjs version has been pushed to 4.0.0, and 3.3.2 has been released as an exact copy of 3.2.0.
